driver-mct/main/prep_ocn_mod.F90

```diff
@@ -382,6 +385,13 @@ subroutine prep_ocn_init(infodata, atm_c2_ocn, atm_c2_ice, ice_c2_ocn, rof_c2_oc
           call seq_map_init_rcfile(mapper_Sw2o, wav(1), ocn(1), &
                'seq_maps.rc', 'wav2ocn_smapname:', 'wav2ocn_smaptype:',samegrid_ow, &
                'mapper_Sw2o initialization')
+          if (iamroot_CPLID) then
+             write(logunit,*) ' '
+             write(logunit,F00) 'Initializing mapper_Fw2o'
+          end if
+          call seq_map_init_rcfile(mapper_Fw2o, wav(1), ocn(1), &
+               'seq_maps.rc', 'wav2ocn_fmapname:', 'wav2ocn_fmaptype:',samegrid_ow, &
+               'mapper_Fw2o initialization')
```

Comment:
Perhaps we should leave out the flux mapping changes until the air-sea coupling PR, otherwise they will be going in untested here.

Reply:
yes - we can make this change in the Momentum coupling PR coming later.